src/lib/sensor.js

```diff
@@ -0,0 +1,38 @@
+const Subdevice = require('./subdevice')
+
+class Sensor extends Subdevice {
+  constructor (opts) {
+    super({ sid: opts.sid, type: 'sensor' })
+  }
+
+  _handleState (state) {
+    super._handleState(state)
+
+    // all fields come at once at first but one-by-one later
+    if (state.temperature) {
+      this._temperature = state.temperature / 100
+    }
+    if (state.humidity) {
+      this._humidity = state.humidity / 100
+    }
+
+    if (this._timeout) {
+      clearTimeout(this._timeout)
+    }
+
+    this._timeout = setTimeout(() => {
```

Comment:
Because the messages come in short bursts I found it annoying to get two messages that had one field with the same value and the other with a slightly different value.
I could of course handle it in the upper layer and maybe even emit different events form here?